### DISCUSSION TITLE
replacing mapbox with maplibre

### DISCUSSION THREAD

**Discussion by BioMarek** [uber/react-map-gl#1466] [Q&A]
Hello

I'm trying to replace mapbox with maplibre, according to this [guide](https://visgl.github.io/react-map-gl/docs/get-started/get-started) adding 'mapbox-gl': 'maplibre-gl' to webpack.config.js should do the trick. But in my application I still see a message informing me that "A valid API token is required to use Mapbox". Webpack indeed replaces some 'mapbox-gl' references with 'maplibre-gl' but react-map-gl still seems to use mapbox. I tried versions 6.1.14 and 5.3.14. Are there any additional steps I need to do, apart from those in the guide above, to replace mapbox with maplibre? Or what could be causing the problem? Thanks for the answers.

**Comment by Pessimistress** [comment]
If you are still using a style that reference tiles on the Mapbox server, then you need a Mapbox token. Otherwise, can you share your package.json and webpack.config.js?

**Reply by BioMarek** [reply]
Yes it was wrong style, respectively I used no style and default one was Mapbox style. Thanks for help.
